File: DSA/Gautam_Shyamnani_590013078/Day29Question1.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter a number: ";
    cin >> n;

    
    if (n & 1) {
        cout << "Odd" << endl;
    } else {
        cout << "Even" << endl;
    }

    return 0;
}

File: DSA/NiharikaChand_590017697/Day29Question1.java
import java.util.*;
class Day29Question1 
{
    public static void check(int n) 
    {
        if ((n & 1) == 0) 
        {
            System.out.println("Output: Even");
        } 
        else 
        {
            System.out.println("Output: Odd");
        }
    }

    public static void main(String args[]) 
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();

        check(n);
    }
}
